app/controllers/slack_controller.rb

```diff
@@ -40,6 +42,8 @@ def verify_token!
   end
 
   def slack_params
+    params[:slack][:token] = params[:token] if params[:token] && !params[:slack][:token]
```

Comment:
```suggestion
    params[:slack][:token] ||= params[:token]
```

Reply:
When there is no `params[:slack][:token]` set, ||= to `params[:token]`. If `params[:token]` is also not set, it retains the same value of `nil` as if it was unset, saving the complex conditional trailing the assignment